Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,19 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Hide Objects Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		for (const SkipEntry& entry : m_LocalCoreStartupParameter.object_removal_codes)
+		{
+			if (!memcmp(src.GetPointer(), entry.data(), entry.size()))
```

Comment:
I would combine the number of vertices and hash as an identifier. You can first check if the number of vertices matches and then check the hash. This way you only hash objects that could possibly match at all.